packages/neuron-ui/src/locales/zh.json

```diff
@@ -232,7 +232,9 @@
       "remove-receiving-address-msg": "确定要删除所选收款地址？",
       "locktime-notice-content": "锁定时间根据区块链实际运行情况会有一定的误差。",
       "release-on": "锁定至",
-      "locktime-warning": "请确保对方钱包支持到期解锁功能。（注：1.交易所一般不支持到期解锁功能 {{extraNote}}）"
+      "locktime-warning": "请确保对方钱包支持到期解锁功能。（注：1.交易所一般不支持到期解锁功能 {{extraNote}}）",
+      "allow-use-sent-cell": "允许使用待处理交易的 Cells 构建交易",
```

Comment:
`outputs` also may be hard and too technical for users.

How about

`Unconfirmed transactions are allowed in this transaction.`

Besides,It occurs to me that we may need to add a note to this option.

`Check this box to build chained transactions to reduce transaction fees and increase transaction speed.`

Reply:
But the expression "transaction in a transaction" is a bit weird.

I mentioned "outputs" because it's directly displayed in the translation detail windows in Neuron.

Reply:
Ok, then the prompt text about the options should be updated to 

`Unconfirmed outputs are allowed in this transaction.`

`允许使用未确认的 outputs 构建交易.`

Reply:
I would suggest using "待确认" instead of "未确认", "待确认" is less fraught.